src/components/HomePage/SearchResultsArea/Offer/OfferDetails.js

```diff
@@ -71,7 +72,7 @@ const OfferDetails = ({
                                 isPage ?
                                     offer.title
                                     :
-                                    <Link href={`/offer/${offer._id}`} className={classes.offerTitleLink}>
+                                    <Link to={`/offer/${offer._id}`} component={RouterLink} className={classes.offerTitleLink}>
```

Comment:
Why did you need the RouterLink?

Reply:
I replaced `href` with `to` and when I clicked in it, nothing happened. Also hovering was not showing the correct link. I saw this `component router link` somewhere in the code and it worked, not really sure why